web/src/app/models/client/Gender.ts

```diff
@@ -0,0 +1,11 @@
+export const Gender = {
+  FEMALE: "Female",
+  MALE: "Male",
+  OTHER: "Other",
+  UNKNOWN: "Unkown"
```

Comment:
Please fix typo "Unkown". Thank you! @chenhuyr

Reply:
Good catch. Fixed.